Products/ZenUI3/browser/resources/js/zenoss/Reports.js

```diff
@@ -26,6 +26,7 @@ report_panel = Ext.create('Ext.panel.Panel', {
     }, {
         xtype: 'panel',
         ref: 'graph_reports',
+        graphs_added: true,
```

Comment:
Okay, so _while being added_ this is `false`. When graphs are already loaded or are not going to be loaded at all, it will be `true`. And analogically to my other comment regarding `processed: true`.

Did I got it right?